collection-scripts/gather_priority_and_fairness

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# We don't want to collect /debug/api_priority_and_fairness in oc adm inspect,
+# https://github.com/openshift/oc/blob/master/pkg/cli/admin/inspect/pod.go#L58-L62
+# because the tokens could be used in a replay attack,
+# thus the route we are taking here is forced by this.
+
+# Safeguards
+set -o nounset
+set -o errexit
+set -o pipefail
+
+BASE_COLLECTION_PATH="/must-gather"
+NAMESPACE_PATH="namespaces/openshift-kube-apiserver"
+POD_PATH="kube-apiserver/kube-apiserver/api_priority_and_fairness"
+APF_PATH="${BASE_COLLECTION_PATH}/${NAMESPACE_PATH}"
+
+# If the cluster version is < 4.6 skip this test
+
+# If cluster version is >=4.6 then the COMPARE will always == 4.6
+#  For example cluster version 4.7:
+#  VERSION=4.7
+#  COMPARE=$(echo -e "4.6\n${VERSION}" | sort -V | head -n1)
+#  echo $COMPARE
+#  4.6
+
+# If cluster version is <4.6 then the COMPARE will be the always the cluster version.
+#  VERSION=4.4
+#  COMPARE=$(echo -e "4.6\n${VERSION}" | sort -V | head -n1)
+#  echo $COMPARE
+#  4.4
+
+VERSION=$(oc get clusterversion version -o jsonpath='{.status.desired.version}')
+COMPARE=$(echo -e "4.6\n${VERSION}" | sort -V | head -n1)
+if [[ ${COMPARE} != "4.6" ]]; then
+  echo "INFO: Cluster version is ${VERSION}. Skipping."
+  exit 0
+fi
+
+# Fetches the POD_NAME where app label is apiserver.
+# Example:
+#  oc get po -n openshift-kube-apiserver -l apiserver -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}'
+#  kube-apiserver-master-0.example.com
+#  kube-apiserver-master-1.example.com
+#  kube-apiserver-master-2.example.com
+KUBE_API_SERVERS=$(oc get po -n openshift-kube-apiserver -l apiserver -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}')
```

Comment:
@zerodayz
the ranges are not using a correct syntax and are missing an `{end}` element.

Please see:
- https://kubernetes.io/docs/reference/kubectl/jsonpath/
- https://bugzilla.redhat.com/show_bug.cgi?id=1900755#c16